tests/base/develop_pipelines.py

```diff
@@ -117,3 +126,21 @@ def run_prediction(dataloader, trained_model, dp=False, min_acc=0.50):
         acc = acc.item()
 
     assert acc >= min_acc, f"This model is expected to get > {min_acc} in test set (it got {acc})"
+
+
+@run_prediction.register(BoringModel)
```

Comment:
this makes the simple model complicated, lest move all these to another model

Reply:
I agree! We can add different dispatcher for more complicated models (e.g., models specifically for dp testing)

Reply:
any change here?